projects/angular/src/emphasis/alert/_alert.clarity.scss

```
@@ -371,6 +392,40 @@
         );
       }
     }
+
+    &.alert-lightweight {
+      background: none;
+      border: none;
+      .alert-items {
+        padding: 0;
+      }
+      .alert-item {
+        min-height: 0;
+      }
+      .close {
+        width: auto;
+      }
```

I would prefer close button to have `display: none` since lightweight alerts don't support close buttons.

```suggestion
      .close {
        display: none;
      }
```